extensions/agent/src/data/pickScheduleData.ts

```
@@ -20,12 +20,18 @@ export class PickScheduleData implements IAgentDialogData {
 		this.jobName = jobName;
 	}
 
-	public async initialize() {
+	public async initialize(): Promise<azdata.AgentJobScheduleInfo[]> {
 		let agentService = await AgentUtils.getAgentService();
-		let result = await agentService.getJobSchedules(this.ownerUri);
-		if (result && result.success) {
-			this.schedules = result.schedules;
+		try {
+			let result = await agentService.getJobSchedules(this.ownerUri);
+			if (result && result.success) {
```

>result [](start = 7, length = 6)

If the result isn't a success we should throw an error